recipes/r-wcorr/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = '1.9.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-wcorr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/wCorr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/wCorr/wCorr_{{ version }}.tar.gz
+  sha256: 53511837df722537edccd2e91ccd9c6f5b7bfc741b19ae1d405801b8426898d8
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-minqa
+    - r-mnormt
+    - libopenblas  # [osx or linux]
```

Comment:
hm. I think that should be `libblas` ... to support switching: https://conda-forge.org/docs/maintainer/knowledge_base.html#switching-blas-implementation

Reply:
awesome, thanks for the tip. I'll also see about making that change over here too: https://github.com/conda-forge/r-npflow-feedstock/pull/1